Gems/WhiteBox/Code/Source/SubComponentModes/EditorWhiteBoxDefaultMode.cpp

```diff
@@ -44,6 +48,10 @@ namespace WhiteBox
     static const char* const HideEdgeUndoRedoDesc = "Hide an edge to merge two connected polygons together";
     static const char* const HideVertexUndoRedoDesc = "Hide a vertex to merge two connected edges together";
 
+    static constexpr AZStd::string_view EditorMainWindowActionContextIdentifier = "o3de.context.editor.mainwindow";
```

Comment:
question: Does this constant/string not exist elsewhere in some more central place? Do we need to create it again here?

Reply:
Same for some of the other strings below too

Reply:
I have a task set up to centralize these in one or more headers, but since I'm working on these migrations and sometimes they get parallelized I'm gonna do that pass all at once at the end

Reply:
Okay got you makes sense, thanks for clarifying